### DISCUSSION TITLE
Loki: Suggest Instant query us user hits error - e.g. limit 500

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#61556] [Observability]
Suggest switching from Range to Instant query for Loki, after parsing and aggregations, as it tends to get high cardinality. A nice suggestion in the error logs (when the limit 500 is hit for example) would be great.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82807. Feel free to continue the discussion around this there. Thank you!
